flume/conf/flume.conf.template

```diff
@@ -29,5 +29,11 @@ CONFIG_FOLDER=/usr/cygnus/conf
 # Which is the config file
 CONFIG_FILE=/usr/cygnus/conf/cygnus.conf
 
-# Name of the agent
+# Name of the agent. This value should match with the begining of the keys in ${CONFIG_FILE} otherwise service not start.
```

Comment:
Fix in [e46fe68](https://github.com/telefonicaid/fiware-connectors/commit/e46fe6894611cc927e1916e003da3d9a9b39a8ef)